Sources/StreamVideoSwiftUI/Utils/PiPUtils.swift

```diff
@@ -0,0 +1,139 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import WebRTC
+import AVKit
+import CoreMedia
+import CoreVideo
+import StreamVideo
+
+extension CMSampleBuffer {
+    
+    static func from(_ pixelBuffer: CVPixelBuffer) -> CMSampleBuffer? {
```

Comment:
nit: Could we make this one a convenience initializer?

Reply:
would prefer to keep this one in a static method

Reply:
Is there a specific reason for it? Convenience initializers are more Swifty.

Reply:
This whole code is not Swifty at all 😄 That's why I was thinking to better keep it as pure input/output method. I can change it if you have strong preference.

Reply:
I don't have strong preference :) thanks for clarifying your thinking.